docs/getting-started/user-guides/decrediton-setup.md

```diff
@@ -0,0 +1,54 @@
+# **Decrediton Setup Guide**
+
+Last updated for v0.8.2.
+
+This guide is intended to help you setup the `Decrediton` application. 
+
+**Prerequisites:**
+
+- Use the [Decrediton Install Guide](/getting-started/install-guide.md#decrediton) to install `Decrediton`.
+
+---
+
+`Decrediton` is a graphical user interface for `dcrwallet`. When this application launches, it automatically starts its own instance of `dcrd` and `dcrwallet` in the background - it will not open if there is already a running instance of `dcrd`. Please note that as of 0.8.2, this application is a beta release - there are many bugs. 
+
+NOTE: At any point in the use of Decrediton, it may become unresponsive or get stuck on a loading screen This can usually be fixed with a restart of the application.
```

Comment:
Not a review question really, but have we ever seen it become unresponsive?  I know it gets stuck on the loading screen but I haven't seen any straight up unresponsive out of it.  If anyone has hit that I'd love to know so I can try to reproduce/fix.